src/backend/routers/post.py

```diff
@@ -94,11 +96,12 @@ async def search_posts(
     return Page(data=posts, count=len(posts), cursor_id=None)
 
 
-@router.get("/{id}")
-async def get_post(id: str) -> PostDetails:
+@router.get("/{post_id}")
+async def get_post(post_id: str) -> PostDetails:
     """Get full details of a specific post."""
     post = await Post.prisma().find_first(
-        where={"id": id, "deleted": False}, include={"tags": True, "media": True, "author": True}
+        where={"id": post_id, "deleted": False},
+        include={"tags": True, "media": True, "author": True},
     )
     if not post:
         raise HTTPException(404, "Post not found")
```

Comment:
`id` needs to be changed to `post_id` on [line 111](https://github.com/idkbrowby/gamingrealm-backend/blob/anand/rating-endpoints/src/backend/routers/post.py#L111)